BSP/Simulator_V2/Src/BSP_CAN.c

```diff
@@ -1,25 +1,35 @@
 /* Copyright (c) 2022 UT Longhorn Racing Solar */
 
 #include "BSP_CAN.h"
-#include "simulator_conf.h"
+#include "Simulator.h"
```

Comment:
Write a test file for this to see if it actually works.

Reply:
Will do